zcash_client_sqlite/src/lib.rs

```diff
@@ -951,6 +952,21 @@ impl BlockSource for FsBlockDb {
     }
 }
 
+#[cfg(feature = "unstable")]
+impl std::fmt::Display for FsBlockDbError {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match self {
+            FsBlockDbError::Fs(io_error) => write!(f, "FsBlockDbError::Fs({})", io_error),
+            FsBlockDbError::Db(e) => write!(f, "FsBlockDbError::Db({})", e),
+            FsBlockDbError::Protobuf(e) => write!(f, "FsBlockDbError::Protobuf({})", e),
+            FsBlockDbError::MissingBlockPath(_) => write!(f, "FsBlockDbError::MissingBlockPath"),
+            FsBlockDbError::InvalidBlockstoreRoot(_) => write!(f, "FsBlockDbError::InvalidBlockstoreRoot"),
+            FsBlockDbError::InvalidBlockPath(_) => write!(f, "FsBlockDbError::InvalidBlockPath"),
+            FsBlockDbError::CorruptedData(e) => write!(f, "FsBlockDbError::CorruptedData({})", e),
```

Comment:
These are not suitable descriptions for a `Display` impl, as they should be user-meaningful (vs `Debug` impls for which these would be fine). See https://github.com/zcash/librustzcash/blob/125d2bc3d58c543ad58eb53b6c65c0d616688f52/zcash_client_sqlite/src/error.rs#L98-L128 for a (terse but sufficient) example of an error `Display` impl.